src/card.css

```diff
@@ -0,0 +1,120 @@
+.card {
+  display: flex;
+  flex-grow: 0;
+  width: var(--card-width);
```

Comment:
do not set external geometry to bem blocks, use bem mixes instead

Reply:
not fixed, move width into bem element